package-lock.json

```diff
@@ -8592,7 +8592,7 @@
       }
     },
     "semantic-release": {
-      "version": "github:dcodeIO/semantic-release#5367f573d21743be81ae1190877a8315405fd364",
+      "version": "github:dcodeIO/semantic-release#3cec5c74de58a4e539d792c807e82ee4500cf28a",
```

Comment:
This change is surprising, in that `5367f573d21743be81ae1190877a8315405fd364` already was the [latest commit](https://github.com/dcodeIO/semantic-release/commits/master).

Reply:
really strange. Reverted this change